examples/produce_v3/javascript/main.js

```diff
@@ -0,0 +1,52 @@
+// if the endpoint is https, please use https package
+const http = require('http');
+
+// handling streaming response
+var recordReceiptCounter = 1;
+callback = function(response) {
```

Comment:
More of a [nit] 

I think it can be changed to the modern ECMASscript (starting from 6+), afaik, function usage is considered as an "old style" : ) 

So, something like this? 

```const callback = (response) => {
    response.on('data', (data) => {
        const chunk = data.toString('utf8');
        if (chunk.trim()) {
            console.log(`Receipt for record #${recordReceiptCounter} is`);
            console.log(JSON.parse(chunk));
            recordReceiptCounter += 1;
        }
    });

    response.on('end', () => {
        console.log("Stream is closed!");
    });
};```